R/translations.R

```diff
@@ -463,6 +502,17 @@ translations_indirect_base <- list(
       stop <- pmax(as.integer(len) + start - 1L, 0L)
       eval(substitute(quote(substr(x, start, stop))))
     }
+  },
+  charindex = function(string, substring) {
+    warning("Using CHARINDEX with non-ASCII characters may return incorrect results due to multiple ways to represent the same character")
```

Comment:
Please use `call. = FALSE` in all calls to `warning()` (this is the usual practice in tidyverse packages; it prevents unhelpful call info from being included in the warning message)